docs/benefits.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: WAVS benefits
+---
+<!--import { Callout } from 'fumadocs-ui/components/callout';-->
+<!--import { Tab, Tabs } from "fumadocs-ui/components/tabs";-->
```

Comment:
this is just cosmetic, turns out it needs to use `//`  instead? (fix script too) or maybe that is just a github thing and it is fine. I dont see issue, just highlighting